docs/nightly/en/getting-started/installation/greptimedb-cluster.md

```diff
@@ -8,7 +8,7 @@ The GreptimeDB cluster can run in the [cluster](/contributor-guide/overview.md)
 
 ```
 curl -fsSL \
-  https://raw.githubusercontent.com/greptimeteam/gtctl/develop/hack/install.sh |\
+  https://raw.githubusercontent.com/GreptimeTeam/gtctl/<%helm-version%>/hack/install.sh |\
```

Comment:
I think `nightly` should keep using the `nightly` build version. But it will introduce more complexity when migrating documents.

Reply:
If the installation script `install.sh` supports version value `nightly` that would be great.  @zyy17

Reply:
The `gtctl` doesn't support specifying the version in the URL.

I plan to re-write the Cluster section by replacing the `gtctl` with docker-compose. The `gtctl` is still a dev tool for deploying binary, which is not recommended for deploying the cluster for a quick start.

@nicecui